partner_supplierinfo/readme/USAGE.rst

```diff
@@ -0,0 +1,15 @@
+To gather a contact's specific supplier reference you'll need a third module that calls the field with the proper context:
+
+  .. code-block:: python
+
+    partner.with_context(supplier_id=supplier.id).supplier_ref
+
+If no supplier is found in the related `partner.supplierinfo` records, we'll get the regular `ref` field content.
+
+To add supplier informations to your contacts:
+  #. Go to any partner form.
+  #. In `Supplier Information` tab customer references can be created edited or consulted.
+    #. In a tree view, the following fields are displayed:
+
+      * `Ref` for the customer reference
+      * `Supplier` for the supplier it belongs to
```

Comment:
The format is still looking a little bit weird: https://github.com/OCA/partner-contact/blob/1fe105ca11464f2a4922616824e6d1e69c58b29e/partner_supplierinfo/readme/USAGE.rst

Reply:
I have rechecked and simplified the formatting

Reply:
Usage and configuration steps should be put with numbered list format (using `#. `). The problem was to not respect the blank line between the previous paragraph and the start of the numbered list, and to have 2 different levels of indentation.

It should be:

```
To add the supplier information to your contacts:

#. Go to any partner form.
#. In the `Supplier Information` tab, the customer references can be created edited or displayed.
#. In the tree view, the following fields are displayed:

   * `Ref` for the customer reference.
   * `Supplier` for the supplier it belongs to.
```

About the nomenclature, it should be `Ref.` (with dot), and use `Vendor` instead of `Supplier`, as Odoo uses it on the rest of the system.

Reply:
OK, I take note, the nomenclature is one of the things left to check, thank you.